### PR TITLE
Added Legal mentions and dynamic statistics

### PR DESCRIPTION
**Pull request by theoPARENTIT**
Created Controller for legal Mentions.
+ some minor modifications

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -12,18 +13,23 @@
...
 
     #[Route('/', name: 'home.index')]
-    public function index(ChartBuilderInterface $chartBuilder): Response
+    public function index(ChartBuilderInterface $chartBuilder, StatisticRepository $repo): Response
     {
         $chart1 = $chartBuilder->createChart(Chart::TYPE_LINE);
+        $dateArray = [];
+        for ($i = 11; $i >= 0; $i-- ) {
+            $dateArray[] = date("F-Y", strtotime("-". $i . " months"));
+            dd($repo->getCountByDate(date("F-Y", strtotime("-". $i . " months"))));
```
Comment (theoPARENTIT) [inline src/Controller/HomeController.php]: DD perdu

File: src/Controller/PolicyController.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PolicyController extends AbstractController
+{
+    #[Route('/policy', name: 'app_policy')]
+    public function index(): Response
+    {
+        return $this->render('policy/policy.html.twig', [
+            'controller_name' => 'PolicyController',
```
Comment (theoPARENTIT) [inline src/Controller/PolicyController.php]: Render sans params, controller_name inutile

**Review by theoPARENTIT — COMMENTED** [review]
Quelques fixes à clear

**Review by matvki — APPROVED** [review]
ok

**Pull request merged**